xml/ay_firewall.xml

```diff
@@ -340,6 +340,21 @@
     </informaltable>
    </sect2>
 
+   <sect2 xml:id="CreateProfile-firewall-stage">
+      <title>When the Firewall Profile is Applied</title>
+
+      <para>
+       The profile is completely applied at the end of the first stage of installation, except when 
+       the installation runs over the network, for example over SSH or VNC. In this case we have 
+       to guarantee that the installed system stays accessible for the whole installation process. For 
+       that reason the firewall configuration is postponed to the second stage. If you need the network 
+       for some reason to fulfill installation successfully, you have to take care of proper firewall 
```

Comment:
hello @mchf, thank for you this PR! Very much appreciated. I have a question or two-- "you have to take care of proper firewall configuration" I am not sure what this means. Does this mean the admin should set up their firewall configuration after installation? Or does this mean the admin must be sure that their profile is correct?

Reply:
It means that if for some reason - e.g. in a  custom script - is network required to fulfill installation, then it is responsibility of user to configure firewall accordingly - we are not able to detect that as we do in case of ssh / vnc installation.